scripts/travis/deploy/publish.sh

```diff
@@ -1,26 +1,27 @@
-#!/usr/bin/env bash
+#!/bin/bash -e
 
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-PROJECT_AFFECTED="$1"
-DOMAIN="$2"
-USERNAME="$3"
-PASSWORD="$4"
-
-cd $DIR/../../../
+DOMAIN="$1"
+USERNAME="$2"
+PASSWORD="$3"
+TAG_VERSION="$4"
+BRANCH_NAME="$5"
+DRY_RUN="$6"
 
 npm ci && npm run build.release
 
-# Get Tag Image
-TAG_VERSION=$(./scripts/travis/deploy/get-docker-image-tag-name.sh)
 echo "Running the docker with tag" $TAG_VERSION
-DOCKER_PROJECT_ARGS="PROJECT_NAME=$PROJECT_AFFECTED"
+DOCKER_PROJECT_ARGS="PROJECT_NAME=content-ce"
 DOCKER_REPOSITORY="$DOMAIN/$REPO_SLUG"
 
 # Publish Image to quay.io or dockerhub or another domain - only publish the version on master - elsewhere version and branch
 if [[ $TRAVIS_BRANCH == "master" ]]; then
```

Comment:
we have still lot of references to $TRAVIS_BRANCH that is probably missing in GHA